### PR TITLE
Move to pkg:dart_flutter_team_lints, require Dart 2.19

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Review by brianquinlan — CHANGES_REQUESTED** [review]

**Review by brianquinlan — CHANGES_REQUESTED** [review]
I don't see an updated to the cupertino_http CHANGELOG.md.

**Review by brianquinlan — APPROVED** [review]

File: pkgs/cupertino_http/example/integration_test/error_test.dart
Comment (zander40) [inline pkgs/cupertino_http/example/integration_test/error_test.dart:1]: https://github.com/dart-lang/http/pull/883

File: pkgs/cupertino_http/pubspec.yaml [hunk 1]
```diff
@@ -2,11 +2,11 @@ name: cupertino_http
 description: >
```
Comment (zander40) [inline pkgs/cupertino_http/pubspec.yaml:2]: pkgs/cupertino_http/lib/src/utils.dart

File: pkgs/cupertino_http/example/integration_test/error_test.dart
Comment (zander40) [reply pkgs/cupertino_http/example/integration_test/error_test.dart:1]: pkgs/cupertino_http/lib/src/utils.dart

**Pull request merged**